packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -230,6 +231,14 @@ export class ProjectDataSource {
     this._specs = specs
   }
 
+  get runAllSpecs () {
+    return this._runAllSpecs
+  }
+
+  set runAllSpecs (runAllSpecs: string[]) {
```

Comment:
Seems this might be more understandable as a method. 

```js
setRunAllSpecs
```

`ctx.setRunAllSpecs(...)` seems more preferrable that `ctx.runAllSpecs = ...`. Right now you have a private var, but you basically make it public with the `set runAllSpecs`.

Reply:
Maybe

```js
#runAllSpecs: string[] = []

get runAllSpecs () {
  return this.#runAllSpecs
}

setRunAllSpecs(specs) {
  this.#runAllSpecs = specs
}
```

Reply:
I like it!